referenceframe/model.go

```diff
@@ -240,3 +240,32 @@ func floatsToString(inputs []Input) string {
 	}
 	return string(b)
 }
+
+// Create an ordered list of transforms given a parent mapping, keeping an eye out for a sentinel string (World).
+func sortTransforms(unsorted map[string]Frame, parentMap map[string]string, start, finish string) ([]Frame, error) {
```

Comment:
To make sure I understand, the goal is to produce an ordered list of transforms starting from `finish` (exclusive) to `unordered[start]` (inclusive), using the parent mapping. This list may not include all of the transforms in `unsorted`. Is that correct?

Reply:
More or less, that is correct. I would defer to @biotinker on the detail of whether or not the intention is to hit all `unsorted` transforms. I don't believe we support branching like that in `ModelJSON`.

Reply:
Agreed. That shouldn't be possible with our current usage, but since this is now a function that can be called from elsewhere, we should start making guarantees.

Reply:
DATA-877 captures the concerns in this comment chain.